common/supplemental/supplementalData.xml

```diff
@@ -545,6 +545,7 @@ The printed version of ISO-4217:2001
             <currency iso4217="HNL" from="1926-04-03"/>
         </region>
         <region iso3166="HR">
+			<currency iso4217="EUR" from="2023-01-01"/>
```

Comment:
Please indent consistently. The next line uses spaces.

Reply:
done